ecosystem/sep-0024.md

```diff
@@ -516,19 +602,21 @@ The fields below should be placed in the request body using the `multipart/form-
+| `asset_code`        | string                                  | Code of the asset the user wants to withdraw. The value passed must match one of the codes listed in the [/info](#info) response's withdraw object. When `quote_id` is specified, `asset_code` must match the quote's `sell_asset` asset code.                                                                                                                                                                                                                                                 |
+| `asset_issuer`      | string                                  | (optional) The issuer of the stellar asset the user wants to withdraw with the anchor. If `asset_issuer` is not provided, the anchor should use the asset issued by themselves as described in their TOML file. When `quote_id` is specified, `asset_issuer` must match the quote's `buy_asset` asset issuer or be omitted.                                                                                                                                                                    |
+| `destination_asset` | string in [Asset Identification Format] | (optional) The asset user wants to receive. It's an off-chain or fiat asset. If this is not provided, it will be collected in the interactive flow. When `quote_id` is specified, this parameter must match the quote's `buy_asset` asset code or be omitted.                                                                                                                                                                                                                                  |
+| `amount`            | number                                  | (optional) Amount of asset requested to withdraw. If this is not provided it will be collected in the interactive flow. When `qoute_id` is specified, this parameter must match the quote's `quote.sell_amount` or be omitted.                                                                                                                                                                                                                                                                 |
+| `quote_id`          | string                                  | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this parameter is provided and the user delivers the deposit funds to the Anchor before the quote expiration, the Anchor must respect the conversion rate agreed in that quote. When `quote_id` is set, `asset_code`, `destination_asset` and `amount` must be validated by the anchor, if present. In case of a conflict with the ones used to create the [SEP-38] quote, this request should be rejected with a `400`. |
```

Comment:
@Ifropc I think you updated the `quote_id` description here on withdrawal by removing the "mandatory" part but did not make the same update for deposit